### PR TITLE
MSTest sample Selenium 3

### PR DESCRIPTION
**Pull request by Baaryan**

### CODE REVIEW

File: MSTest-browserstack/ParallelTest.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    [DataRow("windows", "chrome")]
+    [DataRow("mac", "safari")]
+    public void SearchBstackDemo(string profile, string environment)
+    {
+        var singleTest = new SingleTest(profile, environment);
+        singleTest.Init();
+        singleTest.SearchBstackDemo();
+        singleTest.Cleanup();
+    }
+}
```
Comment (kamal-kaur04) [inline MSTest-browserstack/ParallelTest.cs]: new line

File: MSTest-browserstack/local.conf.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+﻿{
+  "capabilities": {
+    "os_version": "14",
+    "device": "iPhone 12",
+    "realMobile": "true",
+    "browserstack.local": "true",
+    "build": "browserstack-build-1",
+    "name": "BStack test",
+    "browserstack.source": "mstest:sample"
```
Comment (kamal-kaur04) [inline MSTest-browserstack/local.conf.json]: Please follow the correct convention, you can check nunit non-sdk branch

File: MSTest-browserstack/SingleTest.cs [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        Console.WriteLine(driver?.Title);
+        string? productOnPageText = driver?.FindElement(By.XPath("//*[@id=\"1\"]/p")).Text;
+
+        driver?.FindElement(By.XPath("//*[@id=\"1\"]/div[4]")).Click();
+        bool? cartOpened = driver?.FindElement(By.XPath("//*[@class=\"float-cart__content\"]")).Displayed;
+        Assert.AreEqual(cartOpened, true);
+        string? productOnCartText = driver?.FindElement(By.XPath("//*[@id=\"__next\"]/div/div/div[2]/div[2]/div[2]/div/div[3]/p[1]")).Text;
+        Assert.AreEqual(productOnCartText, productOnPageText);
+    }
+}
```
Comment (kamal-kaur04) [inline MSTest-browserstack/SingleTest.cs]: new line

**Review by kamal-kaur04 — CHANGES_REQUESTED** [review]

File: MSTest-browserstack/local.conf.json (hunk 2 above)
Comment (Baaryan) [reply MSTest-browserstack/local.conf.json]: Done

**Review by kamal-kaur04 — APPROVED** [review]

**Review by pranavj1001 — APPROVED** [review]

**Pull request merged**